### PR TITLE
Fix #2762: Removed :subtitled_html_java_proto_lite from the list of exports.

### LINKED ISSUE #2762: Remove :subtitled_html_java_proto_lite from the list of exports in the model library
This task is part of the build system migration from Gradle to Bazel. A prerequesite before working on this issue is that you are able to build Oppia using Bazel. Check out [Oppia Bazel guide](https://github.com/oppia/oppia-android/wiki/Oppia-Bazel-Guide) for Bazel terminology and more on what Bazel does.

## 1. Remove the export

In 

`/oppia-android/model/BUILD.bazel`

Remove :subtitled_html_java_proto_lite from the list of exports in the model library

```BUILD
android_library(
    name = "model",
    visibility = ["//visibility:public"],
    exports = [
        ... # <- Remove it from this list.
    ],
)
```

## 2. Add public visibility to the java_lite_proto_library

Add 

`    visibility = ["//visibility:public"],`

to the java_lite_proto_library that has the name subtitled_html_java_proto_lite.

## 3. Build the app using Bazel and run the tests

Libraries that are indirectly depending on `subtitled_html_java_proto_lite` will now fail to build. Try to build the entire app with Bazel and add the deps dependency

`//model:subtitled_html_java_proto_lite,`

to all the libraries that are now failing to build. You will have to look around for BUILD.bazel files that have libraries that require this replaced dependency.

These libraries will already have the //model dependency and since we removed the :subtitled_html_java_proto_lite from the model library itself, they will need to add the newly available public library directly as a dependency. (Not all libraries with the //model dependency may require the newly available library)

Once the app builds and all tests pass, you're done!

### PR DESCRIPTION
**Pull request by Victor-Titan**
## Explanation

Fixes #2762 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

@FareesHussain PTAL

### CODE REVIEW

**Review by FareesHussain — APPROVED** [review]
LGTM, make this `ready for review` once https://github.com/oppia/oppia-android/pull/2887/checks?check_run_id=2086396253 passes

**Comment by Victor-Titan** [comment]
@FareesHussain I don't understand how there's a build failure if on running the command `bazel build //:oppia` I didn't get any fails. Also, how do I add the dependency to the `//domain:src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputIsGreaterThanRuleClassifierProviderTest` because that is where it's failing. Could you please tell me how to identify a library here?

**Comment by FareesHussain** [comment]
> @FareesHussain I don't understand how there's a build failure if on running the command `bazel build //:oppia` I didn't get any fails. Also, how do I add the dependency to the `//domain:src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputIsGreaterThanRuleClassifierProviderTest` because that is where it's failing. Could you please tell me how to identify a library here?


I guess this is a flaky test

**Review by BenHenning — APPROVED** [review]
Looks like a Bazel example of #2844. We can ignore this flake.

Thanks @Victor-Titan! This LGTM.

File: model/BUILD.bazel [hunk 1]
```diff
@@ -98,6 +98,7 @@ proto_library(
 
 java_lite_proto_library(
     name = "subtitled_html_java_proto_lite",
+    visibility = ["//visibility:public"],
```
Comment (vinitamurthi) [inline model/BUILD.bazel]: Is there a way we can make this a bit more restricted?

Comment (fsharpasharp) [reply model/BUILD.bazel]: I think we could actually remove this line, since the proto is getting indirectly exported through the exploration proto.

Comment (Victor-Titan) [reply model/BUILD.bazel]: I'll remove and build. If it brings errors I'll peruse the visibility folder to select the appropriate visibility for it.

**Comment by Victor-Titan** [comment]
@vinitamurthi @fsharpasharp PTAL
I removed the line pertaining to visibility and the app built successfully.

**Comment by Victor-Titan** [comment]
@vinitamurthi @fsharpasharp PTAL

**Review by fsharpasharp — APPROVED** [review]

**Review by vinitamurthi — APPROVED** [review]
LGTM Thanks!

**Pull request merged**